package.json

```diff
@@ -98,7 +98,7 @@
     "eslint:fix": "eslint . --fix",
     "eslint:check": "eslint . --cache",
     "lint": "npm run eslint:check",
-    "lint:fix": "npm run eslint:fix",
+    "fix": "npm run eslint:fix",
```

Comment:
@padamstx - this change is for you 😂 I remember you recently pointing out our inconsistency between this project and the validator... now, resolved!